test/k8sT/DatapathConfiguration.go

```diff
@@ -497,6 +497,17 @@ var _ = Describe("K8sDatapathConfig", func() {
 			ExpectWithOffset(1, err).Should(BeNil(), "Failure to retrieve host of pod %s", srcPod)
 			// Sanity check
 			ExpectWithOffset(1, srcHost.String()).Should(Equal(k8s1IP))
+			// Fetch srcPod IPv6
+			ciliumPodK8s1, err := kubectl.GetCiliumPodOnNode(helpers.K8s1)
+			ExpectWithOffset(1, err).Should(BeNil(), "Unable to fetch cilium pod on k8s1")
+			endpointIPs := kubectl.CiliumEndpointIPv6(ciliumPodK8s1, "-l k8s:zgroup=testDSClient")
+			// Sanity check
+			ExpectWithOffset(1, len(endpointIPs)).Should(Equal(1), "BUG: more than one DS client on %s", ciliumPodK8s1)
```

Comment:
nit: There is a `HaveLen` Gingko helper: https://onsi.github.io/gomega/#havelencount-int

Reply:
Thanks. Will change it in other WG-related PR.